juniper_official/interface/check-ifl-state-in-out-traffic.rule

```diff
@@ -0,0 +1,425 @@
+/*
+ * Monitors sub interface state and input/output traffic,it notifies when anomalies are found.
+ * Six inputs control detection
+ *
+ *   1) ifd-name, is a regular expression that matches the interfaces
```

Comment:
Observed that interface name variable named in different in all the rules i.e. ifd in one rule, interface-variable in other, interface-name in other, interface-variable-name, etc.. Request you to have consistent name in all rules. Change "interface-name" in all rules

Reply:
Changed variable to interface-name